app/services/idv/flows/in_person_flow.rb

```diff
@@ -5,15 +5,6 @@ module Flows
     class InPersonFlow < Flow::BaseFlow
       attr_reader :idv_session # this is used by DocAuthBaseStep
 
-      STEPS = {
-        state_id: Idv::Steps::InPerson::StateIdStep, # info from state id
```

Comment:
Every instance of `Idv::Steps::InPerson::StateIdStep` should be deleted but I found one in the specs. Take a look at `spec/controllers/idv/cancellations_controller_spec.rb`.